src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Breadcrumb/BitBreadcrumbDemo.razor

```diff
@@ -6,19 +6,22 @@
         <ExamplePreview>
             <div class="example-desc">With items rendered as links</div>
             <div>
-                <BitBreadcrumb Items="GetBreadcrumbItems()"></BitBreadcrumb>
+                <BitBreadcrumb Items="GetBreadcrumbItems()"
+                               OverflowAriaLabel="More links"></BitBreadcrumb>
```

Comment:
is it required to introduce the `OverflowAriaLabel` parameter in each example?

Reply:
@maryamhdr what's your opinion about this issue?

Reply:
> is it required to introduce the `OverflowAriaLabel` parameter in each example?

We can ignore this property for some examples. But I checked the Fluent site. This property was for all examples